app/views/job_postings/index.html.erb

```diff
@@ -1,22 +1,20 @@
 <main style="padding-left: 240px">
   <div class="row">
     <div class="col s12" style="padding-top: 20px;"><h4 class="center-align">Oportunidades laborales</h4></div>
-    <% @job_postings.each do |job_posting| %>
-      <%= link_to job_posting_path(job_posting), :class => "clickableCard" do %>
-        <div class="col s4">
-          <div class="card">
-            <div class="card-image">
-              <%= image_tag job_posting.company.logo, class: "companyLogo" %>
-            </div>
-            <div class="divider"></div>
-            <div class="card-content">
-              <div><strong><%= job_posting.title %></strong></div>
-              <div>Nombre Empresa</div>
-            </div>
-          </div>
+
+    <div class="col s12" style="padding-top: 20px;"><h5>Filtros</h5></div>
+    <%= form_for_filterrific @filterrific do |f| %>
```

Comment:
do this render if you are signed in as a company?

Reply:
It does, I was planning to create a new index just for companies. Since the business logic in the controller and in the view is quite different, I was going to implemented it in a following PR.